10-setup-htcondor.sh

```diff
@@ -133,3 +217,51 @@ echo
 echo "Will use the following token(s):"
 condor_token_list
 echo
+
+cd $LOCAL_DIR
+
+# gwms files in the correct location
+cp -a /gwms/* .
+mkdir -p .gwms.d/bin 
+for target in cleanup  postjob  prejob  setup  setup_singularity; do
+    mkdir -p .gwms.d/exec/$target
+done
+cp -a /usr/sbin/osgvo-singularity-wrapper condor_job_wrapper.sh
+
+# minimum env to get glideinwms scripts to work
+export glidein_config=$LOCAL_DIR/glidein_config
+export condor_vars_file=$LOCAL_DIR/main/condor_vars.lst
+
+# set some defaults for the glideinwms based scripts
+cat >$glidein_config <<EOF
+ADD_CONFIG_LINE_SOURCE $PWD/add_config_line.source
+ALLOW_NONCVMFS_IMAGES True
+CONDOR_VARS_FILE $condor_vars_file
+ERROR_GEN_PATH $PWD/error_gen.sh
+GLIDEIN_SINGULARITY_REQUIRE OPTIONAL
+GLIDEIN_Singularity_Use PREFERRED
+OSG_DEFAULT_CONTAINER_DISTRIBUTION 70%__opensciencegrid/osgvo-el7:latest 30%__opensciencegrid/osgvo-el8:latest
```

Comment:
I assume this is setting the default singularity container, which will get used if singularity is available and the job does not specify an image.

Is there a way to make this configurable?

Reply:
We could expose it as another env var. Or do you have another suggestion on how to interface with container configuration?

Reply:
Yes, env vars work for me.  That would allow me to pick el7 100% of the time.

Reply:
Ok, it is now picked up as an env var. You should be able to do:

``
export OSG_DEFAULT_CONTAINER_DISTRIBUTION="100%__opensciencegrid/osgvo-el7:latest"
```

The image part can be anything available under `/cvmfs/singularity.opensciencegrid.org/`